### PR TITLE
Add test for processing multipart-related emails in fetch_bounces

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
I was trying to replicate an error but didn't manage. Submitting test

### CODE REVIEW

File: tests/phpunit/CRM/Utils/Mail/EmailProcessorTest.php [hunk 1]
```diff
@@ -105,4 +129,27 @@ public function setUpMailing() {
     $this->eventQueue = $this->callAPISuccess('MailingEventQueue', 'get', array('api.MailingEventQueue.create' => array('hash' => 'aaaaaaaaaaaaaaaa')));
   }
 
+  /**
+   * Remove directory, including files within it.
+   *
+   * @param string $dir
+   */
+  protected function rrmdir($dir) {
```
Comment (totten) [inline tests/phpunit/CRM/Utils/Mail/EmailProcessorTest.php]: There's a utility function `CRM_Utils_File::cleanDir($dir, TRUE, FALSE)` which should be pretty similar. I'm not sure if there's some intentional/significant difference here -- but, FWIW, I've used it in other tests (like `CRM_Extension_Manager_ModuleTest`).

**Comment by totten** [comment]
Generally, a big +1 for the idea of adding example email messages to the test suite. This seems pretty important for email-based interactions.

File: tests/phpunit/CRM/Utils/Mail/EmailProcessorTest.php (hunk 1 above)
Comment (eileenmcnaughton) [reply tests/phpunit/CRM/Utils/Mail/EmailProcessorTest.php]: @totten yep - that function worked

**Comment by totten** [comment]
Looks good+sensible.

I pushed a small tweak to the branch. Merge on pass.

**Pull request merged**